pulser/sequence.py

```diff
@@ -13,6 +13,18 @@
 # limitations under the License.
 
 from __future__ import annotations
+from pulser.register import Register
+from pulser.parametrized import Parametrized, Variable
+from pulser.json.utils import obj_to_dict
+from pulser.json.coders import PulserEncoder, PulserDecoder
+from pulser.devices._device_datacls import Device
+from pulser.devices import MockDevice
+from pulser.pulse import Pulse
+from pulser.channels import Channel
+from pulser._seq_drawer import draw_sequence
+import pulser
+from numpy.typing import ArrayLike
+import numpy as np
```

Comment:
Please don't move these imports here. If you want, you can do the conditional imports after these.